File: src/2020/1/js/1.1.js
fs = require("fs");
fs.readFile("src/2020/1/inputs.txt", "utf8", function (err, data) {
  const inputs = data.split("\n");
  console.time("exec");
  for (let i = 0; i < inputs.length; i++) {
    const curr = inputs[i];
    const answer = inputs.find((val) => 2020 - curr - val === 0);
    if (answer) {
      console.log(answer * curr);
      console.timeEnd("exec");
      break;
    }
  }
});

File: src/2020/1/python/1.1.py
import time


with open("src/2020/1/inputs.txt", "r") as f:
    inputs = list(map(int, f.read().splitlines()))
    tic = time.perf_counter()

    for i in inputs:
        for j in inputs:
            if i + j == 2020:
                print(i * j)

    toc = time.perf_counter()
    print(f"Exec in: {(toc - tic)/1000} ms")
